src/resolver.js

```diff
@@ -4,14 +4,17 @@ import {normalizeSwagger} from './helpers'
 
 export function makeFetchJSON(http) {
   return (docPath) => {
-    return http({
+    return Promise.resolve(http({
       url: docPath,
       loadSpec: true,
       headers: {
         Accept: 'application/json'
       }
+    }))
+    .then((res) => {
+      // To allow overriding with spies
```

Comment:
Would be great if this is clearer, overriding what with spies? What spies? Why returning `res.body || res` instead of `res.body` would achieve that?